Source/extensions/processcontainers/implementations/DobbyImplementation/DobbyImplementation.cpp

```diff
@@ -381,10 +381,10 @@ namespace ProcessContainers {
             TRACE(Trace::Error, (_T("Failed to start container %s - internal Dobby error."), _name.c_str()));
             result = false;
         } else {
-            TRACE(ProcessContainers::ProcessContainerization, (_T("started %s container! descriptor: %d"), _name.c_str(), _descriptor));
+            TRACE(Trace::Information, (_T("started %s container! descriptor: %d"), _name.c_str(), _descriptor));
             result = true;
         }
-        _adminLock.UnLock();
+        _adminLock.Unlock();
```

Comment:
ProcessContainers were never compiled for opensource community. So this was not noticed. Thanks for pointing it out